cherab/openadas/repository/atomic.py

```diff
@@ -108,6 +108,47 @@ def update_recombination_rates(rates, repository_path=None):
         _update_and_write_adf11(species, rate_data, path)
 
 
+def add_thermalchargeexchange_rate(species, charge, rate, repository_path=None):
+    """
+    Adds a single thermal charge exchange rate to the repository.
+
+    If adding multiple rates, consider using the update_recombination_rates()
+    function instead. The update function avoids repeatedly opening and closing
+    the rate files.
+
+    :param repository_path:
+    :return:
+    """
+
+    update_thermalchargeexchange_rates({
+        species: {
+            charge: rate
+        }
+    }, repository_path)
+
+
+def update_thermalchargeexchange_rates(rates, repository_path=None):
+    """
+    Thermal charge exchange rate file structure
+
+    /recombination/<species>.json
+
+    File contains multiple rates, indexed by the ion charge state.
+    """
+
+    repository_path = repository_path or DEFAULT_REPOSITORY_PATH
+
+    for species, rate_data in rates.items():
+
+        # sanitise and validate arguments
+        if not isinstance(species, Element):
+            raise TypeError('The species must be an Element object.')
+
+        path = os.path.join(repository_path, 'thermalchargeexchange/{}.json'.format(species.symbol.lower()))
```

Comment:
Change path to be consistent with other CX rate.
https://github.com/cherab/openadas/blob/9379b0651e8e69a3535137bfd959d51cbb44f232/cherab/openadas/repository/beam/cx.py#L112

Reply:
path [changed](https://github.com/Mateasek/cherab_openadas/blob/e9458e956af735110764c7f8d3c7403889e7d9de/cherab/openadas/repository/atomic.py#L149-L150) to: "repository/thermal_cx/donor_element/donor_charge/

``` python 
path = os.path.join(repository_path, 'thermal_cx/{0}/{1}/{2}.json'.format(donor_element.symbol.lower(),
                                                                                          donor_charge, species.symbol.lower()))
```